llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -2342,6 +2343,174 @@ SmallVector<utils::IteratorType> UnPackOp::getLoopIteratorTypes() {
   return iteratorTypes;
 }
 
+//===----------------------------------------------------------------------===//
+// WinogradInputTransformOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult WinogradInputTransformOp::verify() {
+  Operation *op = getOperation();
+  if (getNumInputs() != 1) {
+    return op->emitOpError("expected one input operand");
+  }
+  if (getNumOutputs() != 1) {
+    return op->emitOpError("expected one output operand");
+  }
+  if (!input().getType().isa<ShapedType>()) {
+    return op->emitOpError("expected input type to be shaped");
+  }
+  if (!output().getType().isa<ShapedType>()) {
+    return op->emitOpError("expected output type to be shaped");
+  }
```

Comment:
These checks shouldn't be needed with the ODS being `Variadic<AnyShaped>` I believe.

Reply:
Thanks! Will remove.